package.json

```diff
@@ -81,6 +81,7 @@
     "@types/react": "^16.9.11",
     "@types/react-dom": "^16.9.3",
     "@types/storybook__react": "^4.0.2",
+    "@types/styled-components": "^5.0.1",
```

Comment:
We don't need styled-components itself as a depedency, but we do need the types so that we can import them into the d.ts files otherwise the TypeScript build would fail.

We don't need it as a peerDependency though since it's not required that consumers install the types for grommet to work.